modules/atomic-widgets/styles/style-schema.php

```diff
@@ -152,4 +153,13 @@ private static function get_effects_props() {
 			'box-shadow' => Box_Shadow_Prop_Type::make(),
 		];
 	}
+
+	private static function get_layout_props() {
+		return [
+			'display' => String_Prop_Type::make()->enum([
+				'inline',
```

Comment:
I think you should add all valid values here

/* precomposed values */
display: block;
display: inline;
display: inline-block;
display: flex;
display: inline-flex;
display: grid;
display: inline-grid;
display: flow-root;

/* box generation */
display: none;
display: contents;

Reply:
Done